src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/IUserNotificationServices.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS
+{
+    internal interface IUserNotificationServices
+    {
+        Task<bool> CheckPromptAsync(string promptMessage);
+
+        void NotifyFailureAsync(string renamedString);
```

Comment:
This isn't async, and the parameter name should be changed to better reflect it's purpose.